### ISSUE TITLE
Any plans to port Forms to Vapor 3?

### ISSUE THREAD

**Issue by MrMage** [siemensikkema/forms#25]
Hi,

Last year I built a project that relies on [Forms](https://github.com/nodes-vapor/forms) and [Admin-Panel-Provider](https://github.com/nodes-vapor/admin-panel-provider). I am in the process of porting that project to Vapor 3, but it appears these two library are still fully based on Vapor 2 (which is fairly different).

Do you guys have any plans to port this library to Vapor 3 in the foreseeable future?

Thanks,
Daniel

**Comment by siemensikkema** [comment]
Yes but under new names! See https://github.com/nodes-vapor/admin-panel and https://github.com/nodes-vapor/submissions. Especially the latter is still in need of further work, see: https://github.com/nodes-vapor/submissions/pull/29
